tests/bridges/FacebookBridgeTest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+
+namespace bridges;
+
+require_once __DIR__ . '/../../lib/rssbridge.php';
+require_once __DIR__ . '/../../lib/Configuration.php';
+require_once __DIR__ . '/../../lib/Debug.php';
+require_once __DIR__ . '/../../vendor/simplehtmldom/simple_html_dom.php';
+require_once __DIR__ . '/../../lib/contents.php';
+require_once __DIR__ . '/../../lib/ParameterValidator.php';
+require_once __DIR__ . '/../../lib/BridgeInterface.php';
+require_once __DIR__ . '/../../lib/BridgeAbstract.php';
+require_once __DIR__ . '/../../bridges/FacebookBridge.php';
+
+use PHPUnit\Framework\TestCase;
+
+\Configuration::verifyInstallation();
+\Configuration::loadConfiguration();
+
+class FacebookBridgeTest extends TestCase
```

Comment:
Nice idea about Facebook tests. I suggest to place them to other PR, 'because it is required to discuss it.

Reply:
That makes sense, I'll pull them out elsewhere.
I found them very helpful for development, to make sure things didn't break elsewhere and that they worked in multiple groups. Seems like Travis didn't fail due to them which is good, but I'm not certain whether it actually ran them